weblate/accounts/urls.py

```diff
@@ -84,6 +84,7 @@
     url(r'^remove/$', weblate.accounts.views.user_remove, name='remove'),
     url(r'^confirm/$', weblate.accounts.views.confirm, name='confirm'),
     url(r'^login/$', weblate.accounts.views.WeblateLoginView.as_view(), name='login'),
+    url(r'^metadata/$', weblate.accounts.views.saml_metadata, name='metadata'),    
```

Comment:
```suggestion
    url(r'^saml-metadata.xml$', weblate.accounts.views.saml_metadata, name='saml-metadata'),    
```

So that it's clear that it's saml specific.

Also when saml is enabled, it would be useful to show link to download metadata in management views (probably Tools). But I can add this later if you are not sure how to add it there.

Reply:
I edited urls following your suggestion at https://github.com/WeblateOrg/weblate/pull/3539/commits/e1dc497f7c950b8caddb2d411f8edd07ed71c926
Yes I tested the xml generation and validated it on preposed tools. Seems to work. Tomorrow I'll test the integration with Okta.